src/main/java/com/ericsson/ei/controller/SubscriptionControllerImpl.java

```diff
@@ -45,20 +48,32 @@
 @Api(value = "subscription", description = "The Subscription API for the store and retrieve the subscriptions from the database")
 public class SubscriptionControllerImpl implements SubscriptionController {
 
+    @Value("${ldap.enabled}")
+    private boolean authenticate;
+
     @Autowired
     private ISubscriptionService subscriptionService;
 
     private SubscriptionValidator subscriptionValidator = new SubscriptionValidator();
 
     private static final Logger LOG = LoggerFactory.getLogger(SubscriptionControllerImpl.class);
 
+    // Adding variable(for development purpose) until auth is implemented
+    private String user = "";
```

Comment:
this should not be global. Add it as local variable in the methods where you use them. Remove the comment too.